roles/contiv/templates/contiv.cfg.j2

```diff
@@ -0,0 +1,6 @@
+{
+  "K8S_API_SERVER": "https://{{ hostvars[groups['masters'][0]]['ansible_' + netmaster_interface].ipv4.address }}:{{ kube_master_api_port }}",
```

Comment:
This will just target the ip address of the first master, it may work in most situations but if that api server goes down and there are other api servers still online contiv will fail while the rest of the cluster stays up. Better to point at the cluster hostname. Can you try this?

```
"K8S_API_SERVER": "https://{{ hostvars[groups['masters'][0]].openshift.master.cluster_hostname }}:{{ kube_master_api_port }}",
```